scene/resources/particle_process_material.cpp

```diff
@@ -850,6 +850,27 @@ void ParticleProcessMaterial::_update_shader() {
 	code += "		}\n";
 	code += "	if (RESTART_VELOCITY) {\n";
 	code += "		VELOCITY = get_random_direction_from_spread(alt_seed, spread) * dynamic_params.initial_velocity_multiplier;\n";
+	if (emission_shape == EMISSION_SHAPE_DIRECTED_POINTS) {
+		code += "	int point = min(emission_texture_point_count - 1, int(rand_from_seed(alt_seed) * float(emission_texture_point_count)));\n";
```

Comment:
Reading the code, I think the generated indentation would be wrong, the previous line has two tabs indent, and this one has only one tab.

It seems to be an issue that also exists in some other branches, at least in the section above with the `if (

Reply:
I already have an issue assigned for indentation problems, it's all over the place currently :<

Reply:
In this case the new code should probably be correctly indented from the start at least

Reply:
I'm not sure if the outer indentation is correct

Reply:
You're in a block with the `if` statement, so it should be one more than the current level, and you're inside a method, so that's two levels total

Reply:
You have:
```
void start() {
...
	if (RESTART_VELOCITY) {
		...
	int point = min(emission_texture_point_count - 1, int(rand_from_seed(alt_seed) * float(emission_texture_point_count)));
```
Currently

Reply:
Should be fixed now